### PR TITLE
Dharness/intern experiment

### PR DESCRIPTION
**Pull request by dharness**
I had to modify a couple things on the backend to get things on the right ports/urls etc for the experiment.
@moskalyk 
@nnur

### CODE REVIEW

File: app/services/chatService.ts [hunk 1]
```diff
@@ -48,6 +50,7 @@ module.exports = class ChatService {
 
         // Generate a unique ID for each business:client pair
         let business_socket = this.onlineBusinesses.get(data.businessId);
+        console.log(data.businessId);
```
Comment (moskalyk) [inline app/services/chatService.ts]: Is this still necessary? Or, for debugging measures?

**Comment by moskalyk** [comment]
@dharness LGTM - just console logs, is it good practice to keep in? Or doesn't matter.

File: app/services/chatService.ts (hunk 1 above)
Comment (dharness) [inline app/services/chatService.ts]: Removed.

**Comment by nnur** [comment]
LGTM

**Pull request merged**